src/amplitude-client.js

```diff
@@ -1317,6 +1325,10 @@ AmplitudeClient.prototype._logEvent = function _logEvent(
   }
 };
 
+var _isObservePlanSet = function _isObservePlanSet(scope) {
```

Comment:
Should try to use `let` or `const` where we can

Reply:
yeah, I was wondering why other functions were using var, then just followed, let me update it

Reply:
updated @jooohhn